src/screens/SiteDetailScreen.js

```diff
@@ -0,0 +1,207 @@
+import React, {Component} from 'react';
+import {View, StyleSheet, ScrollView} from 'react-native';
+import {Card, Text, List, ListItem} from 'react-native-elements';
+import {Icon} from 'react-native-elements';
+import {connect} from 'react-redux';
+import {MapView} from 'expo';
+
+const {Marker} = MapView;
+
+import _ from 'lodash';
+
+
+import {linkColorBlue, navyBlueButton} from '../styles/index';
+
+import {navKeys, facilityIconDetails, featureIconDetails} from '../constants';
+import {site_detail_screen, campsite, common} from '../locale.en';
+import {campsiteIcon} from "../styles";
+
+const {header_title} = site_detail_screen;
+const {campsite_form} = campsite;
+const {location} = common;
+
+class SiteDetailScreen extends Component {
+
+    renderFacilities = (facilities) => {
+        return _.map(facilities, (facility, index) => {
+            return (
+                <ListItem
+                    key={index}
+                    title={campsite_form.facilities_options[facility]}
+                    leftIcon={facilityIconDetails[facility]}
+                    hideChevron={true}
+                />
+            );
+        });
+    };
+
+    renderFeatures = (features) => {
+        return _.map(features, (feature, index) => {
+            return (
+                <ListItem
+                    key={index}
+                    title={campsite_form.features_options[feature]}
+                    leftIcon={featureIconDetails[feature]}
+                    hideChevron={true}
+                />
+            );
+        });
```

Comment:
@amgoodman5 for both the features and the facilities I just dropped in `ListItem`s, but I'm not super thrilled with their formatting. Feel free to change the style of anything in this screen.